engine/util/src/main/scala/pl/touk/nussknacker/engine/util/functions/math.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.engine.util.functions
+
+import pl.touk.nussknacker.engine.api.{Documentation, ParamName}
+
+object math {
+
+  @Documentation(description = "Returns the greater of two {@code double} values.")
+  def maxDouble(@ParamName("a") a: Double, @ParamName("b") b: Double): Double =
```

Comment:
Let's make one function with Number as argument and deal with conversions in implementation?